src/ui/wxWidgets/PasswordPolicy.cpp

```diff
@@ -818,3 +853,27 @@ void CPasswordPolicy::OnCopyPassword( wxCommandEvent& WXUNUSED(event) )
     PWSclipboard::GetInstance()->SetData(tostringx(m_passwordCtrl->GetValue()));
   }
 }
+
+/*
+ * Just trying to give the user some visual indication that
+ * the password length has to be bigger than the sum of all
+ * "at least" lengths.  This is not comprehensive & foolproof
+ * since there are far too many ways to make the password length
+ * smaller than the sum of "at least" lengths, to even think of.
+ *
+ * In OnOk(), we just ensure the password length is greater than
+ * the sum of all enabled "at least" lengths.  We have to do this in the
+ * UI, or else password generation crashes
+ */
+void CPasswordPolicy::OnAtLeastChars( wxSpinEvent& WXUNUSED(event) )
```

Comment:
I would change the name to `OnAtLeastPasswordChars` since this handler is specific to that group of controls and is not meant to react to any others.

Reply:
Mmmmmm...ok